### PR TITLE
SLING-8038 - Allow the Repository MOJO specify including features from CLI

### PR DESCRIPTION
**Pull request by simonetripodi**
initial checkin

### CODE REVIEW

**Comment by cziegeler** [comment]
It's a little bit hard to follow what exactly has changed. Maybe you could redo the patch without moving code out of the existing class first. We can later on then rename this class to AbstractXYZ
The new mojo should not be bound to any lifecycle phase, otherwise it's not usable outside of a maven project. 
In addition the only parameter it should get are mvn coordinates for the feature, no includes, no directory etc.
The mvn coordinates can be parsed with the existing code in ArtifactId

**Comment by DominikSuess** [comment]
@simonetripodi @cziegeler - added patches in https://github.com/apache/sling-slingfeature-maven-plugin/pull/14

**Comment by cziegeler** [comment]
This branch has conflicts and I suggest to also update the PR based on my feedback

**Comment by cziegeler** [comment]
Closing this PR as it is outdated
